contributing.md

```diff
@@ -67,7 +67,7 @@ Please note we have a [Code of Conduct](https://www.mozilla.org/en-US/about/gove
 
 The subject should follow the this pattern:
 
-`fixes #github_issue_number type(scope): Description`
+`type(scope): Description`
```

Comment:
It seems useful to continue putting the issue number in the title so that the list of commits has quick links to the fixed issue. Any particular reason not to continue with this?

Reply:
+1 to Mike's comment

Reply:
Oh funnily enough, there's no issue linked to this pr XD  It should still link to the issue even if it's in the commit body instead of the title, @brennie can you file an issue and then format this pr according to the updated formatting? 🙏  That should show that the pr links to the issue the same way.

Reply:
I'm personally not a fan of having fixes in the PR title, just due to the fact that it takes up room and isn't immediately helpful. If I want more information about a commit that has landed, I'm probably going to look at the entire commit message in `git log` or similar. The landed commit also already has the PR number in its summary.

If the above argument doesn't sway folks, I don't feel strongly enough about this issue to debate it further.

Reply:
Oh yeah like, it's not clickable in the actual title, it adds the link elsewhere, and this is clickable in the PR description so you still get the same effect.  I think this is a good change since it accomplishes the effect of linking prs to issues clickably, and also closes the issue which has been broken.

Reply:
@mikewilli are you ok with that?